set-quotas.py

```diff
@@ -0,0 +1,191 @@
+#   Copyright 2016 Massachusetts Open Cloud
+#
+#   Licensed under the Apache License, Version 2.0 (the "License"); you may
+#   not use this file except in compliance with the License. You may obtain
+#   a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#   WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#   License for the specific language governing permissions and limitations
+#   under the License.
+"""Update a project's OpenStack quotas using Google Sheets data
+
+For each entry in the Google Sheet, the script will:
+    - update the specified quotas
+    - generate an email describing the update
+    - send the email to the requestor, cc'ing all project users and anyone
+      else specified in the config file cc_list parameter
+
+Usage:
+    python set-quotas.py
+"""
+import ConfigParser
+from keystoneclient.v3 import client
+from keystoneauth1.identity import v3
+from keystoneauth1 import session
+
+from quotas import QuotaManager
+from message import TemplateMessage
+import spreadsheet
+
+
+class ProjectNotFoundError(Exception):
+    """The specified project does not exist"""
```

Comment:
This works exactly like it should - if a project doesn't exist, it throws an understandable error, rather than just printing out a traceback.  

Eventually I'd like to validate the project names using a dropdown list.  That might be a good idea but we'd have to remember to update it every time a new project was created (plus clean up deleted projects periodically).  But, that makes it easier for people to spam the form - right now, providing a correct (or at least mostly correct) project name serves as a confirmation that it is a legitimate request.  

Thinking "out loud" here - we should require them to input their username as well, and check that the username is actually a member of the project.  

Anyway, stuff like that is certainly on the to-do list for this code, but it's outside the scope of this PR and not high priority.  The high priority right now is to get something in place that the helpdesk folks can use.